File: genealogy-tree-API/GenealogyTree.Domain/DTO/User/UserDetailsModel.cs
using GenealogyTree.Domain.DTO.Person;
using System;
using System.Collections.Generic;

namespace GenealogyTree.Domain.DTO.User
{
    public class UserDetailsModel : PersonDetailsModel
    {
        public ICollection<OccupationModel> Occupations { get; set; }
        public ICollection<EducationModel> Educations { get; set; }
        public string Email { get; set; }
        public string PhoneNumber { get; set; }
        public string About { get; set; }
        public DateTime LastTreeUpdate { get; set; }
    }
}

File: gene-tree/src/app/modules/user/models/user-details.model.ts
import { PersonDetailsModel } from "../../person/models/person-details.model";
import { EducationModel } from "./../education/models/education.model";
import { OccupationModel } from "../occupation/models/occupation.model";

export class UserDetailsModel extends PersonDetailsModel {
  occupations: OccupationModel[];
  educations: EducationModel[];
  email: string;
  phoneNumber: string;
  about: string;
}
